sequence_processing_pipeline/Pipeline.py

```diff
@@ -198,6 +198,31 @@ def _validate_sample_sheet(self, sample_sheet_path):
                                          "n' not found in Bioinformatics secti"
                                          "on"))
 
+            columns_to_sanitize = ['Sample_ID', 'Sample_Name',
```

Comment:
We do _not_ need to change `Sample_ID` - remember Sample_ID can not have . and Sample_Name can not have _

Reply:
Sounds good. I thought there would be some debate over what needed changing. changing '..' to '.' is also purely my own decision; we need to confirm w/the wet-lab that it's needed.